tests/test_all.py

```diff
@@ -340,6 +340,14 @@ def test_no_orphaned_packages(container_per_test: ContainerData) -> None:
             'search-result/solvable-list/solvable[@repository="(System Packages)"]'
         )
     }
+
+    monitoring_stack_packages = {
+        "prometheus",
+        "alertmanager",
+        "blackbox_exporter",
+        "grafana",
+    }
```

Comment:
Sadly, there's more packages pulled in via prometheus & friends that are not in the repo
```suggestion
    monitoring_stack_packages = {
        "prometheus",
        "alertmanager",
        "blackbox_exporter",
        "grafana",
        "system-user-grafana",
        "golang-github-prometheus-alertmanager",
        "system-user-prometheus",
        "prometheus-blackbox_exporter",
    }
```